docs_source_files/content/getting_started_with_webdriver/performing_actions_on_the_aut.nl.md

```diff
@@ -9,7 +9,7 @@ English to Dutch. Do you speak Dutch? Help us to translate
 it by sending us pull requests!
```

Comment:
@cambiph could you please remove the notice from this file since it is already fully translated? Thanks!

Reply:
Done